### PR TITLE
LRQA-68317 Poshi Tests to Validate Input Mask on Integers on Forms

### PR DESCRIPTION
**Pull request by leticiachiu**
Related Issue: https://issues.liferay.com/browse/LRQA-68317

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsNumericField.testcase [hunk 1]
```diff
@@ -800,6 +800,88 @@ definition {
 			fieldValue = "1");
 	}
 
+	@description = "LPS-125861 - Verify that the Input Mask can be enabled on intergers"
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsNumericField.testcase:803]: @leticiachiu, misspelling in word **integers**

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsNumericField.testcase [hunk 2]
```diff
@@ -800,6 +800,88 @@ definition {
...
+			formPageNumber = "0");
+
+		FormFields.viewNumericType(numericType = "integer");
+
+		Form.gotoAdvancedTab();
+
+		FormFields.enableSwitch(fieldName = "Input Mask");
+
+		AssertElementPresent(locator1 = "FormFields#CHARACTER_OPTIONS_SHOW_MORE_BUTTON");
+		Click(locator1 = "FormFields#CHARACTER_OPTIONS_SHOW_MORE_BUTTON");
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsNumericField.testcase:822]: @leticiachiu, can you please add a line break?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsNumericField.testcase [hunk 3]
```diff
@@ -800,6 +800,88 @@ definition {
...
+
+		AssertElementPresent(locator1 = "FormFields#CHARACTER_OPTIONS_SHOW_LESS_BUTTON");
+
+		FormFields.setInputMaskFormat(inputMaskFormat = "E.g +99 (99) 9999-9999");
+
+		FormFields.viewNumericInputMask(inputMaskValue = "E.g +__ (__) ____-____");
+	}
+
+	@description = "LPS-130848 - Verify that the Input Mask can be applied on intergers"
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsNumericField.testcase:831]: @leticiachiu, misspelling in word **integers**

**Review by natocesarrego — CHANGES_REQUESTED** [review]
Terrific pull, @leticiachiu! :smile: :tada: 

Please see my tiny SF comments. Keep this good work and soon I'll jump out from the review of your pulls. :wink:

**Comment by leticiachiu** [comment]
Fixed. Thanksss @natocesarrego! :smiling_face_with_three_hearts::sparkling_heart:

**Comment by natocesarrego** [comment]
ci:forward

**Comment by liferay-forms** [comment]
ci:test:relevant

**Pull request closed without merging**